### ISSUE TITLE
Form definitions: copy-paste these into your form instead of reusing a definition across multiple forms

### ISSUE THREAD

**Issue by alextreme** [open-formulieren/open-forms#4724]
### Thema / Theme

Form designer

### Omschrijving / Description

Discussed with @LaurensBurger . The idea of using formdefinitions to change fields across multiple forms in one go doesn't seem to be the expected approach. Most municipalities trying to use formdefinitions struggle with them and end up having to manage the forms separately.

A better approach in our eyes would be to make use of the form-definitions as if they were form-templates: copy the fields from the formdefinition into your form and then have the ability to modify them further, without impacting other forms.

The suggested approach would be that when you want to make use of a form-definition, that you get the choice between reusing and copying.

**Comment by joeribekker** [comment]
Good idea. I do see this as an addition not as a replacement: If you add a re-usable form and adjust it: Give the option to create  copy at this time, so the changes only affect this form.
